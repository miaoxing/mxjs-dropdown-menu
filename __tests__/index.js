import {render} from '@testing-library/react';
import {DropdownMenu} from '..';

describe('test', () => {
  test('basic', () => {
    const result = render(<DropdownMenu/>);
    expect(result.container).toMatchSnapshot();
  });
});
