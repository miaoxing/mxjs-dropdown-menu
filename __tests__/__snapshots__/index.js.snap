// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test basic 1`] = `
<div>
  <div
    class="filter-dropdown nav-dropdown"
  >
    <ul
      class="nav-tabs tabs-justified border-y"
    >
      <li
        class="dropdown"
      >
        <a
          class="text-active-primary"
          data-target="#search"
          data-toggle="dropdown"
        >
          <span>
            搜索
          </span>
            
          <i
            class="icon small icon-chevron-down"
          />
        </a>
      </li>
    </ul>
    <div
      class="menu-content"
    >
      <div
        class="dropdown-menu fade border-bottom"
        id="search"
      >
        <div
          class="menu sub"
        />
      </div>
    </div>
  </div>
</div>
`;
